src/components/LifeBar.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const LifeBar = ({
+    isLeft,
+    playerPoints
+}) => {
+    const points = playerPoints < 0 ? 0 : playerPoints
+    const barWidth = `${(points * 10) / 5}%`;
+    let color = '#5e8c6a';
```

Comment:
I think, that those colors could be assigned to constants in `constants/colors.js` (in javascript variables). Now, we don't know what the colors are (quite hard to guess even for experiened devs)